src/Microsoft.AspNet.Mvc.Core/Filters/FormatFilter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using Microsoft.Framework.OptionsModel;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace Microsoft.AspNet.Mvc.Core.Filters
```

Comment:
I think this should be in `Microsoft.AspNet.Mvc`. We don't use the 'Core' namespace in the application.